met-web/src/App.tsx

```diff
@@ -94,10 +94,12 @@ const App = () => {
 
     const getTranslationFile = async () => {
         try {
-            const translationFile = await import(`./locales/${language}/${tenant.id}.json`);
```

Comment:
also why the change from tenant.id to basename, this overrides my changes in another PR for the single tenancy/ multi tenancy task